caput/mpiarray.py

```diff
@@ -1476,6 +1465,22 @@ def allgather(self):
 
         return arr
 
+    def copy(self, *args, **kwargs):
+        """Return a copy of the MPIArray.
+        Returns
+        -------
+        arr_copy : MPIArray
+        """
+        return MPIArray.wrap(
+            self.view(np.ndarray).copy(*args, **kwargs), axis=self.axis, comm=self.comm
+        )
+
+    def ravel(self, *args, **kwargs):
```

Comment:
Can you add a docstring here? Just so that if someone only reads the documentation they'll be able to tell that `.ravel` doesn't work, rather than seeing it's presence and inferring that it doesn't work.

Reply:
Oh, good catch.